etc/notebooks/got_scotch_demo/scotch_dashboard.ipynb

```diff
@@ -388,6 +402,22 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "### Radar View\n",
+    "Next, we need to define a RadarView (an inheritance of DOMWidgetView) to watch over the similarities/recommendations table.\n",
```

Comment:
Next, we need to define a `RadarView` that serves as the front-end view for the Python widget class we defined above. This frontend will contain the Javascript that watches for events in the similarities/recommendations table and redraws the radar plot accordingly.